abcclassroom/quickstart.py

```diff
@@ -0,0 +1,105 @@
+"""
+abc-classroom.quickstart
+========================
+
+"""
+
+import os
+from shutil import copy, rmtree
+from argparse import ArgumentParser
+from abcclassroom import __file__
+
+
+def path_to_example(dataset):
+    """
+    Construct a file path to an example dataset.
+    This file defines helper functions to access data files in this directory,
+    to support examples. Adapted from the PySAL package.
+
+    Parameters
+    ----------
+    dataset: string
+        Name of a dataset to access (e.g., "config.yml")
+            
+    Returns
+    -------
+    string
+        A file path (string) to the dataset
+    """
+    abcclassroom_path = os.path.split(__file__)[0]
+    data_dir = os.path.join(abcclassroom_path, "example-data")
+    data_files = os.listdir(data_dir)
+    if dataset not in data_files:
+        raise KeyError(dataset + " not found in abc-classroom example data.")
+    return os.path.join(data_dir, dataset)
+
+
+def create_dir_struct():
+    """
+    Create a directory structure that can be used to start an abc-classroom course. This includes a main directory,
+    two sub directories for templates and cloned files, and a start to a configuration file.
+    """
+    # Creating the default names for the cloned and template repos.
+    course = "course-dir"
+    # Making sure the configuration file is where it's supposed to be.
+    config = path_to_example("config.yml")
+    if not os.path.exists(config):
+        raise ValueError(
+            "config.yml configuration file can't be located, please ensure abc-classroom has been installed correctly"
+        )
+    # Allows users to rename the cloned and template repos.
+    parser = ArgumentParser(description=create_dir_struct.__doc__)
+    parser.add_argument(
```

Comment:
IF we are keeping the name as optional (and I think it should not be optional), should set the default value here so that the user knows what behaviour to expect.

Reply:
i think we should have 2 dirs:

student-cloned-repos
assignment-repos

let's stick with those two being the default name and keep things simple for now. 
but let's use a mandatory course name and org if those are two things that we want users to enter. the only thing is the order and ensuring they don't name the course the org and vice versa.